ktor-features/ktor-websockets/jvm/src/io/ktor/websocket/WebSocketUpgrade.kt

```diff
@@ -71,6 +112,21 @@ public class WebSocketUpgrade(
         return webSocket.coroutineContext[Job]!!
     }
 
+    private fun HeadersBuilder.writeExtensions() {
+        if (!installExtensions) return
+
+        val requestedExtensions = call.request.header(HttpHeaders.SecWebSocketExtensions)
+            ?.let { parseWebSocketExtensions(it) } ?: emptyList()
+
+        val extensions: List<WebSocketExtensionHeader> = extensions.flatMap {
+            it.serverNegotiation(requestedExtensions)
```

Comment:
The extension order is important but by doing this we'll order them in the installation order, right? Shouldn't we keep the client's order or the client should use the order we provided? I don't remember that

Reply:
Yep, it's important and we're keeping the installation order here. Usually, both orders are right, it's up to users to decide which order they prefer